<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageAutomation</class>
 <widget class="QDialog" name="ManageAutomation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Automation Lists</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Add New and Remove Automation Lists</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="autoList">
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="newList">
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::TabFocus</enum>
          </property>
          <property name="toolTip">
           <string>Adds a new (empty) automation list.</string>
          </property>
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeList">
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::TabFocus</enum>
          </property>
          <property name="toolTip">
           <string>Remove selected lists.</string>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="renameList">
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::TabFocus</enum>
          </property>
          <property name="toolTip">
           <string>Rename a selected list.</string>
          </property>
          <property name="text">
           <string>Rename</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editList">
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::TabFocus</enum>
          </property>
          <property name="toolTip">
           <string>Edit selected list</string>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>50</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Assign Lists to Quick Launch Automation Icons</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QGroupBox" name="row1">
     <layout class="QHBoxLayout" name="horizontalLayout1">
      <item alignment="Qt::AlignmentFlag::AlignLeft">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Icon 1:  </string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignLeft">
       <widget class="QComboBox" name="comboBox1">
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::TabFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QGroupBox" name="row2">
     <layout class="QHBoxLayout" name="horizontalLayout2">
      <item alignment="Qt::AlignmentFlag::AlignLeft">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Icon 2:  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox2">
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::TabFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignLeft">
    <widget class="QGroupBox" name="row3">
     <layout class="QHBoxLayout" name="horizontalLayout3">
      <item alignment="Qt::AlignmentFlag::AlignLeft">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Icon 3:  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox3">
        <property name="focusPolicy">
         <enum>Qt::FocusPolicy::TabFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ManageAutomation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
